File: cdk-from-cfn-testing/expected/sam_nodejs_lambda/python/SAMNodeJSLambdaStack.py
from aws_cdk import Stack
import aws_cdk as cdk
import aws_cdk.aws_sam as sam
from constructs import Construct

class SAMNodeJSLambdaStack(Stack):
  def __init__(self, scope: Construct, construct_id: str, **kwargs) -> None:
    super().__init__(scope, construct_id, **kwargs)

    # Transforms
    Stack.add_transform(self, 'AWS::Serverless-2016-10-31')

    # Resources
    myFunction = sam.CfnFunction(self, 'MyFunction',
          runtime = 'nodejs20.x',
          handler = 'index.handler',
          inline_code = 'exports.handler = async (event) => {\n  console.log(event);\n}\n',
        )

File: cdk-from-cfn-testing/expected/sam_nodejs_lambda/csharp/SAMNodeJSLambdaStack.cs
using Amazon.CDK;
using Amazon.CDK.AWS.SAM;
using Constructs;
using System.Collections.Generic;

namespace SAMNodeJSLambdaStack
{
    public class SAMNodeJSLambdaStackProps : StackProps
    {
    }

    public class SAMNodeJSLambdaStack : Stack
    {
        public SAMNodeJSLambdaStack(Construct scope, string id, SAMNodeJSLambdaStackProps props = null) : base(scope, id, props)
        {
            // Transforms
            AddTransform("AWS::Serverless-2016-10-31");

            // Resources
            var myFunction = new CfnFunction(this, "MyFunction", new CfnFunctionProps
            {
                Runtime = "nodejs20.x",
                Handler = "index.handler",
                InlineCode = @"exports.handler = async (event) => {
                  console.log(event);
                }
                ",
            });
        }
    }
}
